lib/galaxy/webapps/galaxy/api/workflows.py

```diff
@@ -136,12 +136,12 @@ def create(self, trans, payload, **kwd):
             'from_history_id',
             'shared_workflow_id',
             'workflow',
-        ] ).intersection( payload )
-        if len( ways_to_create ) == 0:
+        ] )
+        if len( ways_to_create.intersection( payload ) ) == 0:
```

Comment:
Yeah, this is a fix for the deprecated endpoint. The form itself is/will be sending the workflow request to the invocations endpoint: https://github.com/guerler/galaxy/blob/7ffbd0907658ee071fe32393c03bdfee4b6b740c/client/galaxy/scripts/mvc/tool/tool-form-composite.js#L268